tools/config.py

```diff
@@ -114,6 +115,14 @@
 CLEAN_UP_SETTING_TRASH_BIN_ROOT_DIR = 'trash_bin_dir'
 CLEAN_UP_SETTING_MOVED_JOB_DIRS_COMMENT = 'moved_job_dirs_comment'
 
+# definition of values
+JOB_HANDOVER_PROTOCOL_DELAYED_BEGIN = 'job_handover_protocol:delayed_begin'
```

Comment:
I think this should just be 
```suggestion
JOB_HANDOVER_PROTOCOL_DELAYED_BEGIN = 'delayed_begin'
```